wordpress-mobile/WordPress-iOS
# [UI Test] Dismiss image optimization popup

## Description
After introducing the Image Optimization popup https://github.com/wordpress-mobile/WordPress-iOS/pull/21981, `testAddGalleryBlock()` started failing in CI. Actually, only its first run would fail as the popup wouldn't show up in the retry run. 

![image](https://github.com/wordpress-mobile/WordPress-iOS/assets/42008628/7a3b0b37-1970-40fa-926e-c8c3c9bc1ad2)

<img width="300" src="https://github.com/wordpress-mobile/WordPress-iOS/assets/42008628/094cda6a-af03-45ba-b188-64a5270d7b57">

This PR adds the ability to dismiss this popup after adding images to the editor.


## To test
* `testAddGalleryBlock()` should pass in the first try on CI. - Popup is displayed.
* `testAddMediaBlocks()` should pass in the first try on CI. - Popup is NOT displayed.
